src/ansys/aedt/core/hfss3dlayout.py

```diff
@@ -688,14 +688,16 @@ def create_pin_port(self, name, x=0, y=0, rotation=0, top_layer=None, bottom_lay
         else:
             return False
 
-    @pyaedt_function_handler(portname="name")
-    def delete_port(self, name):
+    @pyaedt_function_handler(portname="name", remove_geometry=True)
```

Comment:
```suggestion
    @pyaedt_function_handler(portname="name")
```
I think this is a typo. The function hander is used here to grab old function argument name and log a warning if someone still use them. For example here, previously the argument name was `portname` but now it's is `name`.
If someone uses `portname` a warning asking to use `name` is emited.